pykickstart/commands/network.py

```diff
@@ -824,6 +847,53 @@ def parse(self, args):
 
         return retval
 
+class F38_Network(F27_Network):
+    removedKeywords = F27_Network.removedKeywords
+    removedAttrs = F27_Network.removedAttrs
+
+    def _getParser(self):
+        op = F27_Network._getParser(self)
+        op.add_argument("--ipv4-dns-search", default=None, version=F38, dest="ipv4_dns_search",
+                        help="""
+                        Use this option to set IPv4 search domains.
+                        
+                        Requires ``--device`` to be specified.""")
+        op.add_argument("--ipv6-dns-search", default=None, version=F38, dest="ipv6_dns_search",
+                        help="""
+                        Use this option to set IPv6 search domains.
+
+                        Requires ``--device`` to be specified.""")
+        op.add_argument("--ipv4-ignore-auto-dns", type=ksboolean, default=None, version=F38,
```

Comment:
Hm, I now see you're using ksboolean here. Ends up we really don't use that much (looks like network --onboot and repo --ignoregroups are the only two). It's probably better to just make it a simple bool.